fabric-entity-events-v1/src/main/java/net/fabricmc/fabric/api/entity/event/v1/EntitySleepEvents.java

```diff
@@ -296,6 +296,7 @@
 	public interface ModifySleepingDirection {
 		/**
 		 * Modifies or provides a sleeping direction for a block.
+		 * The sleeping direction is where a player's head is pointing when they're sleeping.
```

Comment:
Does this refer to the first-person perspective direction or the direction the upper half of the player's body is in?

Reply:
There are three possible directions based on this description:

1. The direction in which the upper half of the player's body is facing: ![Upper half of sleeping pose (south)](https://user-images.githubusercontent.com/24855774/131402366-03b354f2-bd45-4b0a-ae2a-dcb6c97c4918.png)
2. The direction the player faces in the first-person perspective: ![First-person sleeping pose (north)](https://user-images.githubusercontent.com/24855774/131402464-7cbd29c2-b3ff-451e-a4f9-ab33bbcf2a13.png)
3. The direction the player faces in a third-person perspective: ![Third-person sleeping pose (up)](https://user-images.githubusercontent.com/24855774/131402386-19979dc4-fdd0-4322-b8ae-c27440eb0569.png)

In addition, the player can face additional directions by entering a bed in a third-person perspective:

Reply:
Would be nice to clarify this a little. like "the position the body is pointing at when looking from feet to head" or so, assuming that's correct.

Reply:
```suggestion
		 * The sleeping direction is where a player's head is pointing from the feet when they're sleeping.
```